payment-manager/src/main/java/com/qpay/paymentmanager/service/impl/PaymentServiceImpl.java

```diff
@@ -41,12 +41,12 @@ public WalletEntity makePayment(final WalletPayment walletPayment) {
         final var updatedToWalletBalance = toWallet.getBalance().add(walletPayment.amount());
         updateBalance(updatedToWalletBalance, toWallet);
 
-        final var notificationCreation = PaymentNotification.builder()
+        final var paymentNotification  = PaymentNotification.builder()
                 .amount(walletPayment.amount())
                 .emailFrom(walletPayment.emailFrom())
                 .emailTo(walletPayment.emailTo())
                 .build();
-        paymentProducer.sendPayment(notificationCreation);
+        paymentNotificationProducer.publishPaymentNotification(paymentNotification );
```

Comment:
too many empty lines in this method. Probably it means this method has a lot of code or different logic and it is hard to read. Most likely this is a reason why you use empty lines. Either remove it or restructure code by extracting code to new methods.